monkey/monkey_island/cc/resources/agent_binaries.py

```diff
@@ -15,7 +15,7 @@ class AgentBinaries(AbstractResource):
     def __init__(self, agent_binary_repository: IAgentBinaryRepository):
         self._agent_binary_repository = agent_binary_repository
 
-    # Used by monkey. can't secure.
+    # Can't be secured, used in manual run commands
```

Comment:
Why we can't secure it ? It is used both in Island and Agent, so at least we can do is add the roles..

Reply:
https://github.com/guardicore/monkey/pull/3203#discussion_r1160014909

Reply:
Ahaaa. Thanks!